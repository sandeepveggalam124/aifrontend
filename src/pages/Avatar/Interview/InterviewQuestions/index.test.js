import { MemoryRouter as Router } from "react-router-dom";
import {render} from "@testing-library/react";
import InterviewQuestion from "./index";


it('renders correctly',()=>{
    const {container}=render(
        <Router>
            <InterviewQuestion/>
        </Router>
    )
    expect(container).toMatchSnapshot()
})