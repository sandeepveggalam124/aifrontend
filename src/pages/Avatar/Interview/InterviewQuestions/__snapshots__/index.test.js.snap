// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    class="row custom-div text-white"
  >
    <div
      class="col col-lg-1 bg-last-image"
    />
    <div
      class="col col-lg-2 bg-black d-flex flex-column justify-content-between p-1"
    >
      <div
        class="border border-opacity-10 rounded-5 w-50 m-2 align-items-center"
      >
        <div
          class="d-flex flex-column justify-content-center align-items-center mt-2"
        >
          <div
            class="d-flex justify-content-center align-items-center"
          >
            <svg
              class="clock-vector-class mx-1"
            >
              clockVector.svg
            </svg>
            <div
              style="font-size: 100%;"
            >
              <span>
                0
              </span>
              :
              <span>
                30
              </span>
              :
              <span>
                0
              </span>
            </div>
          </div>
          <div>
            <p />
          </div>
        </div>
      </div>
      <div
        class="mt-auto"
      >
        <div
          class="d-flex flex-column align-items-center justify-content-around"
        >
          <div
            class="object-fit-lg-cover border rounded"
          >
            <video
              autoplay=""
              height="auto"
              playsinline=""
              width="100%"
            />
          </div>
          <div
            class="p-2"
          />
        </div>
      </div>
    </div>
    <div
      class="col col-lg-6 bg-black d-flex flex-column align-item-stretch p-0"
    />
    <div
      class="col col-lg-2 bg-black d-flex justify-content-center align-items-end mb-2"
    >
      <button
        class="login py-1 justify-content-center align-items-center"
      >
        <a
          class="text-decoration-none finsih-link fs-6 fw-medium"
          href="/interview/finish"
        >
          Next
        </a>
      </button>
    </div>
    <div
      class="col col-lg-1 bg-last-image"
    />
  </div>
</div>
`;
